experiment/build/generate_makefile.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Google LLC
```

Comment:
Probably want a simple test for this. can come in next cl.

Reply:
added TODO